Documentation/security/launch-integrity/secure_launch_overview.rst

```diff
@@ -150,15 +154,15 @@ kernel begins execution.*
  - Kernel boot proceeds normally from this point.
  - During early setup, slaunch_setup() runs to finish some validation
    and setup tasks.
- - The SMP bring up code is modified to wake the waiting APs via the monitor
-   address.
+ - For TXT, the SMP bring up code is modified to wake the waiting APs via the
+   monitor address.
```

Comment:
Do we care about the GETSEC WAKEUP method?

Reply:
I should be clear here. The way you wake the APs after launch is driven by what the ACM tells you to do. After they wake up, we have to put them back in a waiting state. This is done with another mwait/monitor that is unrelated to the TXT/ACM one. When the SMP boot code runs, it finds the right monitor for a given AP by its LAPIC ID and then pokes that monitor. I guess the docs could be made clearer on that point.

Reply:
Ahh okay, the AP wake in in line 142/143 of this file. My bad